tests/codegen/enum/unreachable_enum_default_branch.rs

```diff
@@ -28,13 +28,11 @@ pub fn implicit_match(x: Int) -> bool {
 // The code is from https://github.com/rust-lang/rust/issues/110097.
 // We expect it to generate the same optimized code as a full match.
 // CHECK-LABEL: @if_let(
-// CHECK: start:
-// CHECK-NOT: zext
-// CHECK: select
+// CHECK-NEXT:  start:
 // CHECK-NEXT: insertvalue
 // CHECK-NEXT: insertvalue
 // CHECK-NEXT: ret
 #[no_mangle]
 pub fn if_let(val: Result<i32, ()>) -> Result<i32, ()> {
-    if let Ok(x) = val { Ok(x * 2) } else { Err(()) }
+    if let Ok(x) = val { Ok(x) } else { Err(()) }
```

Comment:
Did you just use revert? This change can retain.

Reply:
Yes, that's what I did for the tests. I can address the tests as suitable.

Reply:
Kept the changes to this file.